tests/judges/test_instruction_judge.py

```diff
@@ -0,0 +1,298 @@
+"""Unit tests for InstructionJudge implementation."""
+
+import unittest
+from unittest import TestCase
+from unittest.mock import Mock, patch
+
+from server.judges.instruction_judge import InstructionJudge
+
+
+class TestInstructionJudge(TestCase):
+    """Test cases for InstructionJudge class."""
+
+    @patch('server.judges.instruction_judge.make_judge')
```

Comment:
Why is mocking necessary here? If it's not can we also audit the remaining tests?